src/BinOp.chpl

```diff
@@ -679,6 +694,23 @@ module BinOp
       var repMsg = "created %s".doFormat(st.attrib(rname));
       return new MsgTuple(repMsg, MsgType.NORMAL);
     }
+    else if (l.etype == bool && val.type == bool) {
+      select op {
+        when ">>" {
+          e.a = l.a:int >> val:int;
+        }
+        when "<<" {
+          e.a = l.a:int << val:int;
+        }
+        otherwise {
+          var errorMsg = notImplementedError(pn,l.dtype,op,dtype);
+          omLogger.error(getModuleName(),getRoutineName(),getLineNumber(),errorMsg);
+          return new MsgTuple(errorMsg, MsgType.ERROR);
+        }
```

Comment:
similar to the other case, we could do something like
```chapel
if val {
  // do shift
}
else {
  e.a = l.a;
}
```
since an array shifted in either direction by `False/0` would just remain the same

Reply:
I made this change.  Thanks for reminding me.